chord_metadata_service/mohpackets/apis/ingestion.py

```diff
@@ -49,103 +49,123 @@
 router = Router()
 
 
-def create_instance(payload, model_cls: Type):
+def create_instances(payloads: List, model_cls: Type):
```

Comment:
this is somewhat picky, but if the payload is a list, then payload does need to be plural

Reply:
Do you want it to be `payload` or `payloads`? I'm confused

Reply:
payload (assuming it is a single List)

Reply:
done